lib/.gitignore

```diff
@@ -0,0 +1,4 @@
+# TODO: remove this file when all files have been replaced with typescript
+configParser.js
```

Comment:
I see, because the published app references lib/file.js in package.json
maybe it should reference built/file.js instead, like Julie suggests